containers/signing-service/install-dependencies.sh.j2

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+
+# Update package lists for Debian and install dependencies
+# Packages needed to build python extensions, dependencies of golem-messages in particular
+readarray dependencies < ${BASH_SOURCE%/*}/packages.txt
+
+echo ${dependencies[@]}
```

Comment:
Maybe also add a message saying that you're printing the dependencies?

Also, won't `apt-get` print them anyway during installation?

Reply:
Done.